ecosystem/nft-metadata-crawler/nft-metadata-crawler-utils/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "nft-metadata-crawler-utils"
```

Comment:
I feel that we keep talking about this. Why are we making this a separate project but the name is very clearly exclusive to metadata crawler?

Reply:
Copying the structure of the utils package in indexer-grpc. Just thought that it might be a good idea to have stuff that would be used by multiple packages (constants, pubsub functions, etc)